src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStrip.java

```diff
@@ -1,189 +1,96 @@
 package frc.trigon.robot.subsystems.ledstrip;
 
-import com.ctre.phoenix.led.*;
-import edu.wpi.first.wpilibj.DriverStation;
-import edu.wpi.first.wpilibj.RobotController;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import edu.wpi.first.wpilibj2.command.button.Trigger;
-import frc.trigon.robot.Robot;
 import frc.trigon.robot.commands.factories.GeneralCommands;
 
-import java.awt.*;
-import java.util.function.Function;
+import java.util.ArrayList;
 
 public class LEDStrip extends SubsystemBase {
-    private static final CANdle CANDLE = LEDStripConstants.CANDLE;
-    private static final Trigger LOW_BATTERY_TRIGGER = new Trigger(() -> !DriverStation.isEnabled() && Robot.IS_REAL && RobotController.getBatteryVoltage() < LEDStripConstants.MINIMUM_BATTERY_VOLTAGE);
-    private static int LAST_CREATED_LED_STRIP_ANIMATION_SLOT = 0;
-    private static int LAST_STRIP_INDEX = 8;
-    private final int animationSlot;
-    private final int offset, numberOfLEDs;
+    public static final ArrayList<LEDStrip> LED_STRIPS = new ArrayList<>();
```

Comment:
Make this an updating array so you won't need the ugly .stream.toarray
Look at the signals code for reference